core/weekly_report_generator.py

```diff
@@ -172,6 +172,8 @@ def generate_weekly_report(
         'meetings_and_activities': format_meetings(calendar_events),
         'google_forms_filled': format_bulleted_list(forms_data, indent="  "),
         'wfo_days': format_weekdays_with_dates(WFO_DAYS),
+        'self_accomplishments': format_bulleted_list(SELF_ACCOMPLISHMENTS),
```

Comment:
```suggestion
        'other_accomplishments': format_bulleted_list(OTHER_ACCOMPLISHMENTS),
```
Gimana kl existing `accomplishments` sekarang kita rename jadi `github`? Biar ga bingung membedakan antara `accomplishments` dan `self_accomplishments`

Reply:
Author jgn lupa bg

Reply:
Eh sori kayake better `github_accomplishments` lebih clear wkwk

Reply:
gas oakwowakokaw

Reply:
done